frontend/scss/components/organisms/release-schedule.scss

```diff
@@ -7,7 +7,12 @@ body.amp-mode-touch .touch-only {
 body.amp-mode-touch .no-touch-only {
   display: none;
 }
-#calendar {
+// The AMP optimizer doesn't support the media attribute, so we hide whichever
```

Comment:
That's unfortunate :(

Reply:
> Not a huge fan of doubling the calendar, but I guess it's a legit workaround for now

My understanding is that's just how it's supposed to be done. I looked to see if we could have the `height` attribute be media-dynamic, but no dice.